### ISSUE TITLE
Usage Statistics for Keptn Installation

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#2310]
Right now there is no way to monitor the usage of a Keptn installation; especially of a large and automated setup. 
For an end-user of Keptn, the following information is interesting:

* Number of projects/services managed by Keptn (**#of service**)
* ~~Number of API calls~~
* Number of CloudEvents (**# of events**)
  * incoming via API (**#of executions**) (--> number of executed sequences)
  * ~~per Keptn service (e.g., how often is jmeter-service being called)~~
  * per project (e.g., some projects require more cloud events than others, ...)
  * per type (e.g., how many deployment-finished events vs. how many tests-triggered events)

Some of this information could be queried from mongodb datastore.

**Tasks:**
- Implement an extra service (in keptn-sandbox) with at `/statistics` endpoint
- The `/statistics` endpoint knows the query parameter `from` and `to` as data, time format (e.g., from=YYYYMMDD and to=YYYYMMDD)

**Note:**
- Long-term this could also be something to include in Keptn Bridge.

**Definition of Done:**
- When calling the endpoint `/statistics`, a JSON is returned showing: 
```
{
  "from": "2020-08-15T08:00Z",
  "to": "2020-09-15T08:00Z",
  
  "projects": [
    { 
      project: "sockshop",
      services: [
        {
          service: "carts",
          exectuedSequences: 7, // number of unique keptn-context 
          "events": {
              "sh.keptn.events.deployment-finished": 6,
              "sh.keptn.event.start-evaluation": 7
           },
           "keptnServiceExecutions": {
               "helm-service": {
                    "sh.keptn.events.deployment-finished": 6,
               },
               "lighthouse-service": {
                    "sh.keptn.events.evaluation-done": 7,
               }
           }
        },
        {
          service: "carts-db",
          exectuedSequences: 2, // number of unique keptn-context 
          "events": {
              "sh.keptn.events.deployment-finished": 2,
              "sh.keptn.event.start-evaluation": 2
           }
           "keptnServiceExecutions": {
               "helm-service": {
                    "sh.keptn.events.deployment-finished": 6,
               },
               "lighthouse-service": {
                    "sh.keptn.events.evaluation-done": 7,
               }
           }
        }
      ]
    }
  ]
}
```

**Comment by christian-kreuzberger-dtx** [comment]
This has been implemented by @bacherfl in https://github.com/keptn-sandbox/statistics-service and has been released as a version 0.1.0 for now (compatible with both, 0.7.1 and the upcoming 0.7.2 version).

I think we should add it to the integrations page here: https://keptn.sh/docs/integrations/#sandbox

**Comment by johannes-b** [comment]
@christian-kreuzberger-dtx can you please file a PR to add the service to the integrations page: https://keptn.sh/docs/integrations/#sandbox
